utils/babeltrace_thapi.in

```diff
@@ -25,7 +29,7 @@ $options_tally = {
   'display_metadata' => [FalseClass, false],
   'display_name_max_size' => [Integer, 80],
   'display_kernel_verbose' => [FalseClass, false],
-  'backend_level' => [String, ''],
+  'backend_level' => [String, '']
```

Comment:
```suggestion
  'backend_level' => [String, ''],
```

Reply:
Rubocopt removed it :( Need to find a special rule to stop doing it

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TrailingCommaInArguments 
I guess one of those

Reply:
Yeah, in this instance, this one:

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TrailingCommaInHashLiteral

Reply:
I would not add it for function parameters though. Only for Array and Hash.

Reply:
And just to be sure `comma` and not `consistent_comma` ?

Reply:
yeah for our use case `comma` seems good enough. I don't think we mix and match.

Reply:
Fixed with e7f54985d4ef5d3c0314776e870203dccb7030e9